python/src/QuantileMatchingFactory_doc.i.in

```diff
@@ -0,0 +1,227 @@
+
+Parameters
+----------
+distribution : :class:`~openturns.Distribution`
+    The distribution defining the parametric model to be adjusted to data.
+    Its parameters define the starting point of the algorithm.
+probabilities : sequence of float, optional
+    The probabilities :math:`p_i \in [0,1]` corresponding to the quantiles.
+    The default value of the list of probabilities defined as follows.
+    We define the interval :math:`[\epsilon, 1 - \epsilon]` where the value of
+    :math:`\epsilon` is defined by the `QuantileMatchingFactory-QuantileEpsilon` key of the 
+    `:class:`~openturns.ResourceMap`.
+    A regular grid :math:`\{p_i\}_{i = 1, ..., K}` of probabilities is defined in the interval :math:`[\epsilon, 1 - \epsilon]`,
+    where :math:`K` is the number of parameters of the distribution.
+    The grid is defined by the equation :math:`p_i = (1 - \rho_i) \epsilon + \rho_i (1 - \epsilon)`
+    where :math:`\rho_i = \frac{i - 1}{K - 1}` for :math:`i = 1, ..., K`. 
+bounds : :class:`~openturns.Interval`, optional
+    Parameter bounds. The default bounds is an empty interval, which implies that the optimization problem is unbounded.
+
+Notes
+-----
+We consider a distribution with :math:`K` parameters. 
+Given a set of probabilities :math:`p_1, ..., p_K \in [0, 1]` 
+and a set of quantiles :math:`q_1, ..., q_K \in \Rset`, 
+we want to estimates the parameters of the distribution such that:
+
+.. math::
+
+    \Prob{X \leq q_i} = p_i
+
+for :math:`i = 1, ..., K`. If a sample is given, then the quantiles are 
+estimated from the data, which leads to the sample 
+quantiles :math:`\hat{q}_1, ..., \hat{q}_K`.
+
+The underlying optimization problem seeks to minimize the sum of slacks between the
+empirical quantiles of the sample and the quantiles of the parametric model:
+
+.. math::
+
+    \Delta = \argmin_{\vect{\theta} \in \Rset^K} \sum_{i=1}^K (\hat{q}_i - q_i)^2
+
+where :math:`K` is the number of parameters of the distribution,
+:math:`p_i` the probabilities,
+and :math:`\hat{q}_i` is the sample quantile and :math:`q_i`
+is the quantile of the parametric distribution
+at the probabilities :math:`p_1, ..., p_K`.
+
+Instead of using a sample, the `buildFromQuantiles()` method can be used if the quantiles are known. 
+This can be useful if some expert knowledge is available. 
+
+See also
+--------
+DistributionFactory
+
+Examples
+--------
+Fit a distribution with 2 parameters. 
+Hence, two quantiles are used to estimate the parameters.
+
+>>> import openturns as ot
+>>> ot.RandomGenerator.SetSeed(0)
+>>> sample = ot.Normal(0.9, 1.7).getSample(10)
+>>> factory = ot.QuantileMatchingFactory(ot.Normal())
+>>> inf_distribution = factory.build(sample)
+>>> print(inf_distribution)
+Normal(mu = 0.267484, sigma = 1.32218)
+>>> print(factory.getProbabilities())
+[0.01,0.99]
+
+We see that the default value of the :math:`\epsilon` parameter is so that we consider the 1% and 99% percentile ranks.
+
+Use 5% and 95% percentile ranks:
```

Comment:
This produces:

![image](https://user-images.githubusercontent.com/31351465/190729466-2a9fbd4e-b14e-4bc8-a6fe-2515b07aa42f.png)

Perhaps a blank line would fix this?